### PR TITLE
feat(swagger): Added a Docker Compose entry for Swagger (AEROGEAR-8548)

### PR DESCRIPTION
**Pull request by craicoverflow**
## Motivation

https://issues.jboss.org/browse/AEROGEAR-8548

## What

Added Swagger container to Docker compose file.

## Why

To make it easier to run and view the API documentation.

## Verification Steps
Add the steps required to check this change. Following an example.
 
1. Run `docker-compose up swagger`
2. Go to localhost:8080
3. Is Swagger running at this address?

## Checklist:

- [x] Code has been tested locally by PR requester
- [ ] Changes have been successfully verified by another team member 

## Progress

- [x] Finished task

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -19,4 +19,10 @@ services:
...
     depends_on:
-      - db
\ No newline at end of file
+      - db
+  swagger:
+    image: swaggerapi/swagger-ui:v3.0.5
+    ports:
+      - 8080:8080
+    environment:
+      API_URL: https://raw.githubusercontent.com/aerogear/mobile-security-service/master/api/swagger.yaml
```
Comment (camilamacedo86) [inline docker-compose.yml]: Could it be changed to get the data from the local file in the dev env? In this way, the UI will reflect the changes performed locally.

Comment (craicoverflow) [reply docker-compose.yml]: I don't believe so. This came up before and we came to the conclusion that it could not be done.

Comment (davidffrench) [reply docker-compose.yml]: Swagger needs the file to be served over http from what I found.

Comment (camilamacedo86) [reply docker-compose.yml]: However, is not possible to retrieve the file by something as localhost:3000/api ...?

Comment (davidffrench) [reply docker-compose.yml]: If there is a web server running locally, that is indeed possible.

**Pull request merged**